server/collaborator/src/minio.ts

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright © 2023 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { WorkspaceId } from '@hcengineering/core'
+import { MinioService } from '@hcengineering/minio'
+
+export class MinioClient {
```

Comment:
May be we should reuse one from minio package and make it API available if it not?

Reply:
Replaced with the on from minio package. I wanted to make slightly more convenient API for the client, but now most of methods not used.